### PR TITLE
Restoring upgrade docs for 0.9

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic/issues/3280 and https://github.com/vmware/vic/issues/4017. This reflects the upgrade behaviour as it is in build 8351.

@emlin, can you please take a look? Thanks!

### CODE REVIEW

**Comment by stuclem** [comment]
Thanks @emlin !

**Comment by stuclem** [comment]
♻️

File: doc/user_doc/vic_admin/upgrade_vch.md [hunk 1]
```diff
@@ -42,10 +50,9 @@ For descriptions of the options that `vic-machine upgrade` includes in addition
 During the upgrade process, `vic-machine upgrade` performs the following operations:
 
 - Validates whether the configuration of the existing VCH is compatible the new version. If not, the upgrade fails.
```
Comment (sgairo) [inline doc/user_doc/vic_admin/upgrade_vch.md]: Should this maybe read "compatible _with_ the new version"?

Comment (stuclem) [inline doc/user_doc/vic_admin/upgrade_vch.md]: Done in https://github.com/vmware/vic/pull/4144/commits/2c07b3d163be8e6e0c9aa3b9d76746eb9c8875bd

**Pull request merged**